tools/samtools.wdl

```diff
@@ -637,3 +652,61 @@ task collate_to_fastq {
         maxRetries: max_retries
     }
 }
+
+task fixmate {
+    meta {
+        description: "This WDL task runs Samtools fixmate on the input BAM file. This fills in mate coordinates and insert size fields."
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file to add mate information. Must be name-sorted or name-collated."
+        prefix: "Prefix for the output file. The extension will be added."
+        extension: "File format extension to use for output file. Allowable types: [.sam, .bam, .cram]"
```

Comment:
IDK how I feel about allowing SAM. IMO we should only be dealing with compressed forms of this data type. SAM just tends to be absurdly large and ultimately impractical.

And by eliminating `.sam`, this can be changed to a Boolean. Makes life a little easier.

Reply:
If you disagree and want to support `.sam`, this line should be updated to use `choices: {}` syntax.